phoebe/atmospheres/passbands.py

```diff
@@ -607,52 +613,52 @@ def load(cls, archive, load_content=True):
                     self.extern_wd_idx = header['wd_idx']
 
                 # Model atmospheres in the passband file:
-                atms = set([entry.split(':')[0] for entry in self.content])
+                stored_atms = set([entry.split(':')[0] for entry in self.content])
 
                 # We have to iterate over available atms rather than stored atms because
                 # the stored atms may not be available in the current version of PHOEBE.
                 available_atms = models.ModelAtmosphere.__subclasses__()
                 for atm in available_atms:
-                    if atm.name not in atms:
+                    if atm.name not in stored_atms:
                         continue
 
                     if atm.external:
                         continue
 
-                    basic_axes = tuple([np.array(list(hdul[f'{atm.prefix}_{name}'].data[name])) for name in atm.basic_axis_names])
+                    basic_axes = tuple(np.asarray(hdul[f'{atm.prefix}_{name}'].data[name]) for name in atm.basic_axis_names)
```

Comment:
this is the change that is breaking interpolation

Reply:
it would be awesome to understand at some point _why_ that breaks it, but I'm too worn out right now to muster the strength for it. I commented out this line and replaced it with the old line as motivation to look into it at some point, but all tests are passing again and I hope we're done with the changes now.